### ISSUE TITLE
Credentialing: If response from referee is not received within X days, close the application

### ISSUE THREAD

**Issue by tompollard** [MIT-LCP/physionet-build#1305]
When credentialing, if a referee has not responded within a week or two, it is unlikely that they will respond at all. In these cases, it is better to close the application than for it remain open in perpetuity (or until the applicant sends a puzzled email). For the new workflow only, we should consider automatically closing applications that have not received a response from the referee within 7 days of the most recent contact date.

**Comment by Lucas-Mc** [comment]
I agree with this and I think something that goes along with it is automatic recontact of the reference... So maybe the process could be:
1. Contact the reference
2. Automatic recontact after 7 days if no response
3. Reject after 7 more days if no response

**Comment by briangow** [comment]
@Lucas-Mc perhaps we could copy the applicant on the email to the reference in step 2.  I'd imagine that some of the time the email could be going to a spam folder for the reference or they just keep skipping over it for some reason.  Letting the applicant also know that we haven't heard back from the reference may initiate them to talk to their reference about this.

**Comment by tompollard** [comment]
> perhaps we could copy the applicant on the email to the reference in step 2

I have mixed feelings about this because it adds pressure on the referee to accept the application. If referee does not support the application, they should feel free to make this decision without burden as far as possible.
